lib/commands/run-script.js

```diff
@@ -1,6 +1,20 @@
-const { log, output } = require('proc-log')
+const { output } = require('proc-log')
 const pkgJson = require('@npmcli/package-json')
 const BaseCommand = require('../base-cmd.js')
+const { getError } = require('../utils/error-message.js')
+const { outputError } = require('../utils/output-error.js')
+
+// TODO this is missing things like prepare, prepublishOnly, and dependencies
+const cmdList = [
```

Comment:
This was intentionally put inline where it was because it was added when we were trying to squeeze any ms we could out of `npm run`.

Reply:
Reverted this.